system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/TestUtils.java

```diff
@@ -19,7 +19,13 @@ private TestUtils() {
     }
 
     /**
-     * The DID that resolves to the sample DID Document in docker compose (served by the nginx container)
+     * The DID that resolves to the sample DID Document for a client in docker compose (served by the nginx container)
      */
-    static final String DID_WEB = "did:web:did-server:test-authority";
+    static final String CLIENT_DID_WEB = "did:web:did-server:test-client";
+
+    /**
+     * The DID that resolves to the sample DID Document fpr the Dataspace Authority in docker compose (served by the nginx container).
+     * Did web format reference: https://w3c-ccg.github.io/did-method-web/#create-register
+     */
+    static final String DATASPACE_DID_WEB = "did:web:localhost%3A8080:test-dataspace-authority";
```

Comment:
```suggestion
    static final String DATASPACE_DID_WEB = "did:web:did-server:test-dataspace-authority";
```

Reply:
The dataspace did is called not from inside of docker but from the outside, that's why I used localhost